README.md

```diff
@@ -1,2 +1,76 @@
-# ts_m2fpga
-M2 Controller FPGA code
+# M2 Cell Controller FPGA Code
+
+## Overview
+
+This project will improve the FPGA code in the [M2 Cell Controller LaVIEW project](https://github.com/lsst-ts/ts_mtm2_cell).
+In that version, the VI named **4-port serial Master Slave.vi**, under `Chassis (cRIO-9038) >> FPGA Target (RIO0, cRIO-9038)` in the Project Explorer, interacts with the hardware inputs and outputs.
+
+## Directory Structure
+
+```bash
+ts_m2fpga
+├── src
+│   ├── m2fpga.lvproj
+│   ├── mainFPGA.vi
+│   ├── moduleRelays.vi
+│   ├── moduleDaq.vi
+│   ├── moduleIlc.vi
+├── doc
+│   ├── versionHistory.md
+│   ├── moduleRelays.md
+│   ├── moduleDaq.md
+│   ├── moduleIlc.md
+├── dependencies
+│   ├── NiFpga_mainFPGA.h
+│   ├── NiFpga_mainFPGA.lvbitx
+│   ├── NiFpga.c
+│   ├── NiFpga.h
+├── README.md
+└── .gitignore
+```
+
+- `src`: in here are all LabVIEW files.
```

Comment:
`src`: contains the LabVIEW FPGA project.

Reply:
Done